com.unity.visualeffectgraph/Editor/Data/VFXDataMesh.cs

```diff
@@ -157,6 +157,10 @@ public Material GetOrCreateMaterial()
             if (subMaskIndex != -1)
                 mappings.Add(new VFXMapping("subMeshMask", subMaskIndex));
 
+            var compiledData = contextToCompiledData[context];
+            compiledData.indexInSystemDesc = outBufferDescs.Count;
```

Comment:
Another typo?

Reply:
Yes, outBufferDescs should be outSystemDesc, I will remove this code anyway.

Reply:
See https://github.com/Unity-Technologies/Graphics/pull/4457#issuecomment-839768246